test/ops_tests.jl

```diff
@@ -0,0 +1,143 @@
+using MLX
+using Random
+using Test
+
+@testset "ops" begin
+    Random.seed!(42)
+
+    element_types = MLX.supported_number_types(MLX.DeviceTypeGPU) # TODO Excluding Float64
+
+    array_sizes = [
+        # (), # TODO: Excluded broadcasting over 0-dimensional Array: Yields scalar result
```

Comment:
Broadcasting over a zero-dimensional array returns a scalar and not a zero-dimensional array in current Julia: https://github.com/JuliaLang/julia/issues/28866